privacy_base/readme/USAGE.rst

```diff
@@ -0,0 +1,2 @@
+In the "Data Protection", open the "Settings" menu to find and enable 
```

Comment:
Shouldn't this be in configuration?

Reply:
This module alone is not provided any feature, they need to be activated in "Settings" (or installing "feature" addon modules), so I think this should be pointed out here.

Reply:
I imagine, but in any case, this is configuration. AFAIK there are addons without usage instructions, just because they have no usage, just configuration.

Well, just my opinion, I'm not gonna debate anymore if you just want it here. :wink:

Reply:
The Usage section is mandatory, and the Configuration section mandatory. So if I have to choose one, it has to be Usage.
Don't worry too much - I'm pretty sure this will evolve and we will end up having both.